staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -167,6 +171,8 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 		if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
 			if ignoreClientCallFailures {
 				klog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
+
+				versionedAttr.Attributes.AddAnnotation(MutationAuditAnnotationPrefix  + MutationAuditAnnotationFailedOpenKeySuffix, "true")
```

Comment:
`MutationAuditAnnotationPrefix` is a domain used for tracking webhook mutation existence ("webhook A was called, but did it mutated the API request or not?") purpose. You can add a separate domain for fail-open annotations

Reply:
Yeah makes sense.  Any suggestions on what domain name to use?  (Not super familiar with the annotation naming convention here).

Reply:
Maybe `fail-open.webhook.admission.k8s.io/`, or `{error/failure}.webhook.admission.k8s.io/` if we want to generalize this annotation in future

Reply:
Picked `<hook-name>.failure.webhook.admission.k8s.io` as the new key.